backend/de.metas.business/src/main/java/de/metas/product/impexp/ProductImportProcess.java

```diff
@@ -103,8 +103,6 @@ protected void updateAndValidateImportRecords()
 		MProductImportTableSqlUpdater.builder()
 				.selection(selection)
 				.ctx(getCtx())
-				.tableName(getImportTableName())
-				.valueName(I_I_Product.COLUMNNAME_Value)
```

Comment:
what's the reason for removing those 2 lines?
why is that considered dead code?